domains/ffmuc25.conf

```diff
@@ -112,49 +107,6 @@
         },
       },
     },
-    bandwidth_limit = {
-      enabled = false,
-      egress = 1500,
-      ingress = 12000,
-    },
-  },
-
```

Comment:
Ist es Absicht das die Autoupdate-Funktion deaktivert wird?

Reply:
Jein. Hiermit wird die Funktion nicht deaktiviert sondern dedupliziert.
Die auto-update config ist in der site.conf und wird für die einzelnen Domains übernommen.
https://github.com/freifunkMUC/site-ffm/blob/5a7f4f60c5899798a007aaf15d3e8ad65b144011/site.conf#L56-L91

Reply:
ich würde dazu raten, aber dann einen key von dir (und vielleicht einem weiteren der heute aktiven freifunker in münchen) hinzuzufügen.

Reply:
Hab ich mal als Kommentar in die PR description geschrieben, damit es nicht untergeht.
Danke für den Hinweis.